package.json

```diff
@@ -29,23 +29,26 @@
     "minify": "uglifyjs dist/bundles/stache.umd.js --screw-ie8 --compress --mangle --comments --output dist/bundles/stache.umd.min.js",
     "test": "skyux test"
   },
+  "peerDependencies": {
```

Comment:
Yep. This module requires a certain module, but won't install it itself. This prevents version collisions when multiple modules need the same module.